### PR TITLE
[ClusterFuzzLite] Support local runs

### PR DESCRIPTION
**Pull request by jonathanmetzman**
Fixes https://github.com/google/clusterfuzzlite/issues/58

### CODE REVIEW

File: infra/cifuzz/filestore/filesystem/__init__.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+    # Make directory and any parents.
+    os.makedirs(filestore_path, exist_ok=True)
+    copied_files = set(dir_util.copy_tree(directory, filestore_path))
+    if not delete:
+      return True
+
+    files_to_delete = initial_files - copied_files
+    for file_path in files_to_delete:
+      os.remove(file_path)
```
Comment (oliverchang) [inline infra/cifuzz/filestore/filesystem/__init__.py:71]: Why not just delete the entire directory first?

**Review by oliverchang — APPROVED** [review]
LGTM! Will you add some documentation for how to actually run this locally from someone's command line?

File: infra/cifuzz/filestore/filesystem/__init__.py (hunk 1 above)
Comment (jonathanmetzman) [reply infra/cifuzz/filestore/filesystem/__init__.py:71]: I can add a comment, but I feel like this approach more closely mimics the behavior of rsync.

Comment (jonathanmetzman) [reply infra/cifuzz/filestore/filesystem/__init__.py:71]: This is mainly for concurrency.

**Pull request merged**